<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProbeSettingStartPage</class>
 <widget class="QWizardPage" name="ProbeSettingStartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>This probe design wizard determines the most settings for designing probes for target and non-target sequences.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Select from the following choices:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton_default">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Use all default settings (and start designing primer now)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_saved">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Use previously saved settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_show">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Show default settings</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_default</sender>
   <signal>clicked()</signal>
   <receiver>ProbeSettingStartPage</receiver>
   <slot>onFinishSetting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_saved</sender>
   <signal>clicked()</signal>
   <receiver>ProbeSettingStartPage</receiver>
   <slot>onFinishSetting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_show</sender>
   <signal>clicked()</signal>
   <receiver>ProbeSettingStartPage</receiver>
   <slot>onContinue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onFinishSetting()</slot>
  <slot>onContinue()</slot>
 </slots>
</ui>
